sentry-tracing/Cargo.toml

```diff
@@ -12,7 +12,8 @@ Sentry integration for tracing and tracing-subscriber crates.
 edition = "2018"
 
 [dependencies]
-sentry-core = { version = "0.23.0", path = "../sentry-core" }
+sentry-core = { version = "0.23.0", path = "../sentry-core", features = ["client"] }
+sentry-backtrace = { version = "0.23.0", path = "../sentry-backtrace" }
```

Comment:
why was this change necessary? I don’t see these imports being used? Please revert this in that case.